server/src/main/java/io/crate/planner/statement/CopyFromPlan.java

```diff
@@ -226,15 +226,11 @@ public static ExecutionPlan planCopyFromExecution(AnalyzedCopyFrom copyFrom,
             clusteredByInputCol = InputColumns.create(table.getReference(clusteredBy), sourceSymbols);
         }
 
-        boolean failFast = boundedCopyFrom.settings().getAsBoolean("fail_fast", false);
-        if (failFast) { // fail_fast should be able to provide meaningful exception messages
-            copyFrom = new AnalyzedCopyFromReturnSummary(copyFrom.tableInfo(), copyFrom.table(), copyFrom.properties(), copyFrom.uri());
-        }
-
```

Comment:
this was not enough, it only worked for failing cases. If no uri had errors, the operation should behave as if no summary was collected.